### ISSUE TITLE
New tabs in OSX

### ISSUE THREAD

**Issue by lewisdonovan** [Swordfish90/cool-retro-term#384]
There doesn't seem to be an option to open a new tab in OSX. I can see the menu option to open the tab bar (Cmd+Shift+T) and this works, but there's still no option to open a fresh tab. I've tried Cmd+T, and double-clicking the tab bar, but neither works. Cheers.

**Comment by strangeattr** [comment]
In fact this surprises me as I didn't knew Cool Retro Term supports tab at all. I could find no way to open a tab, neither in the menu nor on the keyboard. Ctrl+T, Ctrl+Shift+T and Ctrl+N do nothing (I am using a Linux PC, not a Mac).

**Comment by lewisdonovan** [comment]
Yeah, I'm not sure why the tab bar is there, maybe it's just a Mac thing. Either way, it doesn't seem to do anything.

**Comment by strangeattr** [comment]
I have actually opened a ticket asking to add this functionality: https://github.com/Swordfish90/cool-retro-term/issues/421

**Comment by keilmillerjr** [comment]
I agree. I just found this awesome terminal app that would put my owning a single lcd mind at ease. But it can not multitask. Multiple tasking is something I like. I am usually compiling, opening apps, configuring files. cmd + t in the mac terminal works great. Please put this on the todo list and I will recommend this app to as many people as I can!

**Comment by keilmillerjr** [comment]
One other suggestion is to hold the arrow keys to move indicator more than one character.

**Comment by allisonober** [comment]
So if you're looking for multiple terminal support, I found a way to have multiple windows at once (not tabs though.)

It's very limited, but simple to do. In MacOS, I copied the application and renamed it to make it a separate app. Now I have two windows to use, which act like separate applications.

<img width="1096" alt="Screen Shot 2020-10-16 at 10 35 33 AM" src="https://user-images.githubusercontent.com/52665928/96272375-0483e680-0f9c-11eb-9e49-9a7c4d130463.png">

The downsides are configs won't be shared (you'll have to export in one and import in the other to update your profiles), and you'll need to repeat this depending on how many extra windows you want.

The up side is this is a very easy technique that can likely work on any OS. And a bonus, you can cmd+tab between the terminals, which I actually prefer.

(And sorry if I'm spamming this message across issues, just want to notify anyone that desires multiple terminal support)

**Comment by d4nielhdz** [comment]
Hey @allisonober I'm trying to do what you're doing but upon opening the copied app, it just takes me to the instance that's already open.

I'm doing this: 
`cp -r /Applications/cool-retro-term.app /Applications/cool-retro-term-2.app
open /Applications/cool-retro-term-2.app
`

But it only opens the instance that's already running (cool-retro-term.app and not cool-retro-term-2.app)
